### ISSUE TITLE
[FEATURE] - Social Media 

### ISSUE THREAD

**Issue by briandowns** [dictu-lang/Dictu#552]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the solution you'd like

I was hoping to get your (@Jason2605) feedback on the idea of social media advocacy and trying to get larger community usage and involvement. After comparing and contrasting Dictu against a few other similar languages I think Dictu is far beyond the others for a few reasons. Not that stars on repos are a definitive gauge, it'd be cool to get that count up as well as overall adoption.

WIth all of that, I was curious if there has been any thoughts around starting a twitter account or other social account for advocacy around new features, bug fixes, cve remediation, and most important, new releases.

**Comment by Jason2605** [comment]
It's a valid point, and one that I haven't thought of too much previously. In the back of my mind I'm always comparing Dictu to other languages and potentially more importantly their eco-systems. I would absolutely love it if Dictu was to take off and have a few more people using it but at the same time, if that doesn't happen, i'm still really happy with the progress we're bashing out on this.

I personally don't actually use twitter (not for any major reason, just never really got into it) so it's not something I've had a huge amount of thought, but I do now and again shill Dictu to people on other mediums like Discord now and again or the odd dev.to post.

I think there's a couple of things that at the moment really hinder the adoption of Dictu, one being concurrency that we mentioned in the other issue, and the other arguably *huge* issue is the tooling around Dictu. For example, there's no package manager, so distributing and using others code is gonna be an awful user experience (by comparison) and the other is IDE / Text Editor support is almost non-existent (e.g highlighting / autocompletion / debugging / etc). There is the VSc plugin (https://github.com/dictu-lang/DictuVSC) but it needs a ton of work.

To summarise, I think it would be wonderful to be able to push Dictu out there a bit more, but I think there are a couple of major stumbling blocks we would run into that would be hard to answer. What do you think?

**Comment by briandowns** [comment]
I agree 100% with the set of points that might be holding Dictu back at the moment. I think concurrency is the biggest since the other languages in this space have it and next would be a package manager. I was going POC a package manager (from my previously forked version of Dictu) when the PRs I have open get closed.

I use twitter for tech and some other stuff and have been actively pushing dictu when I can with screenshots of example code, new prs merged, etc. Would be happy to continue but would also be happy to start an specific Dictu account and start tweeting things from there. Might get more eyes and more work completed quicker.

**Comment by Jason2605** [comment]
Yeah you’re probably right, and concurrency is definitely of higher priority, although the others are still important. Yeah I do plan on trying to get through your PRs, apologies on the slight delay on them!

Oh that’s awesome, didn’t realise you were doing that thank you! I’ll trust your judgement on whichever you feel is better to gain traction in terms of a user account or a dedicated account, I’m happy for you to start one though if you wish

**Comment by briandowns** [comment]
Cool! I'll figure something out and send you some links. As for the PRs, no rush. There for whenever. :D Thanks!

**Comment by briandowns** [comment]
I created a Slack org for anyone using Dictu that wants to chat about it.

https://join.slack.com/t/slack-50f2911/shared_invite/zt-1m3asjoqe-dmCYzu_GFE3yGF7DmrQJrw
